e2e/tests/ui-driven/src/create-flow-with-geospatial.spec.ts

```diff
@@ -169,7 +176,51 @@ test.describe("Flow creation, publish and preview", () => {
     ).toBeVisible();
     await clickContinue({ page });
 
+    const drawBoundaryTitle = page.getByRole("heading", {
+      name: "Confirm your location plan",
+    });
+    await expect(
+      drawBoundaryTitle,
+      "We are in the Draw Boundary component",
+    ).toBeVisible();
+
+    await checkGeoJsonContent(page, "drawgeojsondata", mockMapGeoJson);
+
+    const area = "490.37";
+
+    await expect(
+      page.getByText(area),
+      "We can see a value for area",
+    ).toBeVisible();
```

Comment:
nit: the component will have calculated the precise area in both metres squared & hectares, but we expect only the _rounded_ metres squared to be visible to the user - can this be simplified to:
```
const area = "The property boundary you have drawn is 490.37 m²",
await expect(page.getByText(area)).toBeVisible();
```
(Again `"We can see"` followed by `.toBeVisible()` feels very redundant to me!)

Reply:
True, I do feel like it does add value with other checks where it may look like duplication. Like when we navigate back somewhere or when we're using a certain value in a certain place, but these do look a bit indulgent